libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -100,6 +102,27 @@ type cgroupData struct {
 	pid       int
 }
 
+// isIgnorableError returns whether err is a permission error (in the loose
+// sense of the word). This includes EROFS (which for an unprivileged user is
+// basically a permission error) and EACCES (for similar reasons) as well as
+// the normal EPERM.
+func isIgnorableError(err error) bool {
```

Comment:
Does this only make sense for rootless container? If so, can we add rootless check in it?

Reply:
The rootless check is already done at the only caller. We could pass `isRootless` as a bool if you think it'll look cleaner, but we can't just do an `os.Geteuid()` here because (in theory) you can run as rootless in other contexts too.

Reply:
> We could pass isRootless as a bool if you think it'll look cleaner

That'll be better, my concern is that people may use this function in other circumstances and that's actually not what they expected.

Reply:
Oh, I forgot to fix this. I'll send a follow-up.